src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer.SymbolStartAnalyzer.cs

```diff
@@ -34,12 +36,14 @@ private sealed partial class SymbolStartAnalyzer
             public SymbolStartAnalyzer(
                 AbstractRemoveUnusedParametersAndValuesDiagnosticAnalyzer compilationAnalyzer,
                 INamedTypeSymbol eventArgsTypeOpt,
-                ImmutableHashSet<INamedTypeSymbol> attributeSetForMethodsToIgnore)
+                ImmutableHashSet<INamedTypeSymbol> attributeSetForMethodsToIgnore,
+                in DeserializationConstructorCheck deserializationConstructorCheck)
```

Comment:
this being 'in' seems unnecessary.  the copy is going to happen below anyways.

Reply:
Doesn't this result in one fewer copies though?

Reply:
i don't see the value in that sort of microoptimization.  Is a copy a problem that needs to be avoided?

Reply:
No, this is called once per compilation. I've removed it.